packages/remix/test/fixtures-vite/01-no-preset/vercel.json

```diff
@@ -0,0 +1,3 @@
+{
+  "framework": "remix"
```

Comment:
Remix + Vite is currently detected as `vite` preset. The changes in this PR fix that, but these changes need to be deployed on the API-side first. So we can remove these `vercel.json` files in the fixtures in a follow-up PR after this PR is deployed.